File: Python/get_aps.py
import requests

url = "https://api.growflux.com/v1/aps"

payload = {}
headers = {
  'Accept': 'application/json',
  'Authorization': 'Bearer {{ insert token here }}'
}

response = requests.request("GET", url, headers=headers, data = payload)

print(response.text.encode('utf8'))

File: javaScript/get_aps.js
var settings = {
    "url": "https://api.growflux.com/v1/aps",
    "method": "GET",
    "timeout": 0,
    "headers": {
      "Accept": "application/json",
      "Authorization": "Bearer {{ insert token here }}"
    },
  };
  
  $.ajax(settings).done(function (response) {
    console.log(response);
  });